### ISSUE TITLE
1.0 To Do

### ISSUE THREAD

**Issue by daviddarnes** [daviddarnes/network-icons#1]
- [ ] Setup some sort of build script for all the partials
- [ ] Setup a local server method for development
- [ ] Style page to be more presentable
- [ ] Add the most common icons that people need
- [ ] Add JavaScript to add and take away icons from the output
- [ ] Sort out the most essential options
- [ ] Add instructions on how to implement
- [ ] Inform @nouveller of this repo

**Comment by nouveller** [comment]
Nice work @daviddarnes, we need to find a day to work on this.

**Comment by daviddarnes** [comment]
Agreed. I think Jekyll might be the best solution for building and local development. All we'd need is the config file and that will be useful for variables. Folder structure could be like this:

- _layouts
- _includes
- _icons
- js
- css

Seems a bit overkill to use it but it'll help us stop repeating code and give a nice simple way to run a local server.

**Comment by nouveller** [comment]
I was just saying Jekyll seems a bit overkill, but yeah it could help stop repeating code. Though I'm not sure how much repeated code we'd have as it's only a one pager.

:smile:

**Comment by daviddarnes** [comment]
Well considering the icons will be repeated at least once (we'll have them shown on the page and in the copy code section) plus the icons appear in a list with a checkbox so that could be a partial too. I was writing the html last night and it was getting tedious to write each li with each icon.

**Comment by nouveller** [comment]
Ha, I can imagine. Sounds good to me then!

**Comment by daviddarnes** [comment]
Cool, plus Jekyll won't add any bloat to the site. Just the config file, which we'll probably need for any repeated lines of text.

**Comment by daviddarnes** [comment]
:up:
